examples/src/routes/MeasureValueFilter.jsx

```diff
@@ -1,20 +1,68 @@
 // (C) 2007-2019 GoodData Corporation
 import React from "react";
 
-import MeasureValueFilterExample from "../components/MeasureValueFilterExample";
 import ExampleWithSource from "../components/utils/ExampleWithSource";
+import MeasureValueFilterExample from "../components/MeasureValueFilterExample";
+import MeasureValueFilterShownInPercentageExample from "../components/MeasureValueFilterShownInPercentageExample";
+import MeasureValueFilterStackedToHundredPercentExample from "../components/MeasureValueFilterStackedToHundredPercentExample";
+import MeasureValueFilterFormattedInPercentageExample from "../components/MeasureValueFilterFormattedInPercentageExample";
 
 import MeasureValueFilterExampleSRC from "!raw-loader!../components/MeasureValueFilterExample"; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import MeasureValueFilterShownInPercentageExampleSRC from "!raw-loader!../components/MeasureValueFilterShownInPercentageExample"; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import MeasureValueFilterStackedToHundredPercentExampleSRC from "!raw-loader!../components/MeasureValueFilterStackedToHundredPercentExample"; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import MeasureValueFilterFormattedInPercentageExampleSRC from "!raw-loader!../components/MeasureValueFilterFormattedInPercentageExample"; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
 
 export const MeasureValueFilter = () => (
     <div>
-        <h1>Filter by Value</h1>
+        <h1>Filter by Measure Value</h1>
         <p>
             Here is how you can filter the entire insight by the value of a measure. The granularity is
             defined by the attributes in the insight.
         </p>
         <hr className="separator" />
-        <ExampleWithSource for={MeasureValueFilterExample} source={MeasureValueFilterExampleSRC} />
+        <h2>Comparison and range filters</h2>
+        <p>Example of filtering of visualization by either one or two measure values.</p>
+        <div className={"s-measure-value-filter-example-1"}>
+            <ExampleWithSource for={MeasureValueFilterExample} source={MeasureValueFilterExampleSRC} />
+        </div>
+        <h2>Filter by measure value shown in %</h2>
+        <p>
+            In the case when visualization is filtered by a measure that is shown in %,{" "}
```

Comment:
I would rephrase it to: When visualization is filtered by a measure that is shown in %, the filter value is in the original measure scale and not in the percentage scale that is displayed in the visualization.